src/Tickets/Commerce/Gateways/PayPal/Assets.php

```diff
@@ -77,6 +77,7 @@ public function register() {
 				'jquery',
 				'tribe-common',
 				'tribe-tickets-commerce-js',
+				'tec-tickets-commerce-notice-js',
```

Comment:
Do we want to use `tec` as the prefix? If we do we should do it for the rest of the assets here. Let's ask Gustavo what he prefers.

Reply:
@bordoni please confirm.